db/carrier_icons.yml

```diff
@@ -2,11 +2,52 @@
   name: ups
   label: UPS
   group: other
+-
+  name: usps
+  label: USPS
+  group: other
+-
+  name: sendle
+  label: Sendle
+  group: other
+  
+-
+  name: poste-italiane
+  label: PosteItaliane
+  group: other
+-
+  name: mondial-relay
+  label: MondialRelay
+  group: other
 -
   name: fedex
-  label: FedEx
+  label: Fedex
+  group: other
+-
+  name: evri
+  label: Evri
+  group: other
+-
+  name: dpd
+  label: Dpd
   group: other
 -
   name: dhl
   label: DHL
   group: other
+-
+  name: correos
+  label: Correos
+  group: other
+-
+  name: colissimo
+  label: Colissimo
+  group: other
+-
+  name: chronopost
```

Comment:
![CleanShot 2024-02-02 at 12 26 49@2x](https://github.com/Checkout-Blocks/shipping_carrier_icons/assets/3484527/8021eb98-4ccf-4e23-8b75-4a164f83f68d)

Like DPD, we can provide a chronopost_symbol.svg with just the chronopost blue box. https://www.dpd.com/cz/wp-content/uploads/sites/226/2020/05/DPD_Logo-manu%C3%A1l.pdf